schema/generate/33-schema_update_35.sql

```diff
@@ -0,0 +1,4 @@
+-- Adding GitLab Platform
+INSERT INTO "augur_data"."platform" ("pltfrm_id", "pltfrm_name", "pltfrm_version", "pltfrm_release_date", "tool_source", "tool_version", "data_source", "data_collection_date") VALUES (25151, 'GitLab', '3', '2020-12-27', 'Manual Entry', 'Sean Goggins', 'GitLab', '2020-12-27 16:07:20');
+--  # Pull request commit updates
+update "augur_operations"."augur_settings" set value = 35 where setting = 'augur_data_version'; 
```

Comment:
This needs to be added to the oneshot schema install script. And probably some other stuff we forgot to add too...

Reply:
ok but that wouldn't prevent this from being merged. Where is the oneshot?

Reply:
Again, this is not resolved. It needs to be in the oneshot as we made that with the intention to keep it up to date with the small update scripts, otherwise we ship 2 different versions of the schema and that's not good for anyone. It's in the `schema/` folder, same as all the other schema files. It's called `20201025-Release-0.14.0.sql`